src/test/Testcafe/wcag_tests/wcag-tests.js

```diff
@@ -0,0 +1,131 @@
+import App from '../app';
+import { Selector, t, ClientFunction } from 'testcafe';
+import axeCheck from 'axe-testcafe';
+import RunTimePage from '../page-objects/runTimePage';
+import { AutoTestUser } from '../TestData';
+import DesignerPage from '../page-objects/designerPage';
+import DashBoard from '../page-objects/DashboardPage';
+
+let app = new App();
+let runtime = new RunTimePage();
+let designerPage = new DesignerPage();
+let dashboard = new DashBoard();
+
+const getLocation = ClientFunction(() => document.location.href);
+
+fixture.only('WCAG 2.0 tests for Altinn Studio')
+	.page(app.baseUrl)
+	.beforeEach(async t => {
+		await t
+			.useRole(AutoTestUser)
+			.resizeWindow(1536, 864)
+	})
+
+	test('Accessibility testing for preview page', async t => {
+		await t
+			.navigateTo(app.baseUrl + 'designer/tdd/wcag#/test')
+			.switchToIframe(runtime.testBrukerIframe)
+			.expect(runtime.testUsers[0].exists).ok()
+			.hover(runtime.testUsers[0])
+			.click(runtime.testUsers[0])
+			.expect(runtime.startNewButton.exists).ok()
+			.click(runtime.startNewButton)
+			.switchToMainWindow()
+			.expect(runtime.testUserHeader[0].exists).ok()
+		const { error, violations } = await axeCheck(t);
+    await t.expect(violations.length === 0).ok(createReport(violations));
+	});
+
+	test('Accessibility testing for deployment to test environment page', async t => {
+		await t
+			.navigateTo(app.baseUrl + 'designer/tdd/wcag#/test')
+			.click(designerPage.testeNavigationTab)
+			.hover(designerPage.leftDrawerMenu)
+			.click(designerPage.testeLeftMenuItems[1])
+		const { error, violations } = await axeCheck(t);
+		await t.expect(violations.length === 0).ok(createReport(violations));	
+	});
+
+	test('Accessibility testing for party selection page', async t => {
+    await t
+      .navigateTo(app.baseUrl + 'designer/tdd/wcag#/test')
+      .switchToIframe(runtime.testBrukerIframe)
+      .expect(runtime.testUsers[1].exists).ok()
+      .hover(runtime.testUsers[1])
+      .click(runtime.testUsers[1])
+      .expect(runtime.startNewButton.exists).ok()
+      .click(runtime.startNewButton)
+      .switchToMainWindow()      
+			const { error, violations } = await axeCheck(t);
+			await t.expect(violations.length === 0).ok(createReport(violations));	
+		});
+
+	test.skip('Accessibility testing for designer page', async t => {
```

Comment:
If this test is not valid anymore, can it be removed ?

Reply:
Added navigation to drag and drop section of the designer. Removed skip clause.